src/jdk.jdwp.agent/share/native/libjdwp/commonRef.c

```diff
@@ -522,7 +529,7 @@ commonRef_pin(jlong id)
         } else {
             jobject strongRef;
 
-            strongRef = strengthenNode(env, node);
+            strongRef = strengthenNode(env, node, JNI_FALSE);
```

Comment:
Can you comment the JNI_FALSE parameter in these calls please e.g.
```
strongRef = strengthenNode(env, node, JNI_FALSE /* not pinAll */);
```

Reply:
ok

Reply:
I though normally we just gave the name of the argument in the comment, and not apply the value. So in the above case we would use `JNI_FALSE /* pinAll */`.

Reply:
AFAIAA both styles are used in different places, so whichever you prefer.